Amplify/Categories/Auth/AuthCategoryBehavior.swift

```diff
@@ -125,8 +125,34 @@ public protocol AuthCategoryBehavior {
                                listener: AuthConfirmForgotPasswordOperation.EventListener?) ->
     AuthConfirmForgotPasswordOperation
 
-    func changePassword(currentPassword: String,
-                        newPassword: String,
-                        options: AuthChangePasswordOperation.Request.Options?,
-                        listener: AuthChangePasswordOperation.EventListener?) -> AuthChangePasswordOperation
+    // MARK: - User Management
```

Comment:
Yes, I was debating whether to add it or not. I think it is okay to add since it doesnot make a difference to the end user. Just makes our code organized and clean. Moved to AuthCategoryUserBehavior